online_pmi

```diff
@@ -2,21 +2,23 @@
 
 """Align words based on stepwise EM alignments with PMI scores."""
 
-from collections import defaultdict
+from collections import defaultdict, Counter
 import itertools as it
 import sys
 import igraph, utils
```

Comment:
Yes, the imports need improvement. I just thought it more important to feed back fixes to the bugs I introduced before messing with new stuff.